src/app/shared/page/footer/footer.component.ts

```diff
@@ -0,0 +1,68 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-footer',
+  templateUrl: './footer.component.html',
+  styles: [
+    `
+      @media (min-width: 768px) {
+        .text-center {
+          text-align: right !important;
+        }
+      }
+    `,
+  ],
+})
+export class FooterComponent {
+  infoLinks = [
+    { title: 'معلومات عن شغل', route: '/routePath' },
+    { title: 'الأسئلة الشائعة', route: '/routePath' },
+    { title: 'ضمان حقوقك', route: '/routePath' },
+    { title: 'شروط الاستخدام', route: '/terms-of-use' },
+  ];
+
+  otherLinks = [
+    { title: 'الشركاء', route: '/routePath' },
+    { title: 'المقالات', route: '/routePath' },
+    { title: 'مركز المساعدة', route: '/routePath' },
+  ];
+
+  pageLinks = [
+    { title: 'تسجيل جديد', route: '/routePath' },
+    { title: 'قدم كشريك', route: '/routePath' },
+    { title: 'تصفح كل الفئات', route: '/routePath' },
+    { title: 'اتصل بنا', route: '/routePath' },
+  ];
+
+  appLinks = [
+    {
+      imgSrc: 'assets/images/app-store.png',
+      imgAlt: 'app-store',
+      route: '/routePath',
```

Comment:
use array routes

```suggestion
      route: ['', 'routePath'],
```

Reply:
what is the difference? I searched about differences but didn't fully understand